### ISSUE TITLE
When in Xinput Linux mode, Steam and all steam games do not recognise gamepad inputs.

### ISSUE THREAD

**Issue by DAMO238** [inputlabs/organization#78]
Setting the mode to Xinput Linux, Console, Console Legacy and FPS Fusion are all affected. In Steam and any Steam game, all gamepad inputs are disabled. This means that the following does not work:

* Console: Everything except mouse
* Console Legacy: Everything
* FPS Fusion: Left stick

Games tested in Steam: 
* ELDEN RING
* Deadlink
* Risk of Rain 2
* Cyberpunk 2077 (as a non-Steam game opened via Steam)

Outside of Steam, games handle inputs just fine.

Games tested outside Steam: 
* Cyberpunk 2077
* HTML 5 controller tester.

I have tried enabling and disabling steam input, to no avail. Switching into Generic controller mode shows some things working. All the buttons are in a different order and need to be reconfigured to make sense. In Steam, I put all the buttons in the right place, and I was able to move around the steam UI using gamepad inputs, however, as soon as you go into game, these settings seem to be lost.

**Comment by marcos-diaz** [comment]
Thanks for the report, which Linux distro?

**Comment by DAMO238** [comment]
Arch, although I think I have a workaround. I disabled xboxdrv and used xpad (from linux-lts) instead, which seems to work just fine.

**Comment by marcos-diaz** [comment]
Yes, we test it with `xpad` which is what most distros have by default.
